### PR TITLE
Vtol improvements

### PR DESCRIPTION
**Pull request by hamishwillee**
This is first step in improving vtol docs. What this does is 
- explain what VTOL means
- shows/explains the difference between the types at high level.
- Introduces the flight modes
- Provides links to info on assembly and config. 

Following this, I figure I'll do more specific pages for each of the VTOL types. They would be similar - what I'm hoping to do is eventually build this up so that someone could enter from any of the VTOL pages and find out what they type of frames do, and how to construct them. At the moment it is a bit artificial because I have never built a VTOL. 

@sfuhrer @RomanBapst  Would love your review please.

### CODE REVIEW

File: en/frames_vtol/README.md [hunk 1]
```diff
@@ -1,20 +1,127 @@
...
+  margin-left: auto;
+  padding: 5px;
+}
+
+.wrapper img {
+  margin-top: 10px;
+}
+</style>
+
+<div class="wrapper">
```
Comment (hamishwillee) [inline en/frames_vtol/README.md:47]: Note this renders nicely in vuepress but not github. Sorry!

Comment (hamishwillee) [reply en/frames_vtol/README.md:46]: It looks like this
image

File: en/frames_vtol/README.md [hunk 2]
```diff
@@ -1,20 +1,156 @@
...
+
+  <div class="grid_item">
+    <div class="grid_item_heading">Tailsitter</div>
+    <div class="grid_text">
+    Rotors permanently in fixed wing-position.
+    Takes off and lands on tail. Flips to fixed wing orientation for normal flight.
```
Comment (sfuhrer) [inline en/frames_vtol/README.md:52]:
```suggestion
    Takes off and lands on tail. Whole vehicle tilts forward to enter forward flight.

```

File: en/frames_vtol/README.md [hunk 3]
```diff
@@ -1,20 +1,156 @@
...
+    <img src="../../assets/airframes/vtol/wingtraone/hero.jpg" title="wingtraone" />
+    <ul>
+      <li>Simple and robust</li>
```
Comment (sfuhrer) [inline en/frames_vtol/README.md:55]:
```suggestion
      <li>Simple and robust, allows for compact design</li>

```

File: en/frames_vtol/README.md [hunk 4]
```diff
@@ -1,20 +1,156 @@
...
+      <li>Fewer actuators</li>
```
Comment (sfuhrer) [inline en/frames_vtol/README.md:56]:
```suggestion
      <li>Minimal set of actuators</li>

```

File: en/frames_vtol/README.md [hunk 5]
```diff
@@ -1,20 +1,156 @@
...
+      <li>Can be hard to control, particularly in wind</li>
+      <li>Poor hovering</li>
```
Comment (sfuhrer) [inline en/frames_vtol/README.md:58]:
```suggestion
      <li>Tradeoff between efficiency in hover and forward flight, as same actuators are used</li>

```

File: en/frames_vtol/README.md [hunk 6]
```diff
@@ -1,20 +1,156 @@
...
+  </div>
+
+<div class="grid_item">
+  <div class="grid_item_heading">Tiltrotor</div>
+  Rotors swivel 90 degrees to transition from multicopter to forward flight orientation.
+  Takes off and lands on belly.
+  <div class="grid_text">
+  <img src="../../assets/airframes/vtol/eflite_convergence_pixfalcon/hero.jpg" title="Eflight Confvergence" />
+  <ul>
+    <li>Mechanically more complex with additional actuators</li>
```
Comment (sfuhrer) [inline en/frames_vtol/README.md:70]:
```suggestion
    <li>Additional actuators for motor tilts</li>
    <li>Mechanically complex tilting mechanism</li>

```

File: en/frames_vtol/README.md [hunk 7]
```diff
@@ -1,20 +1,156 @@
...
+    <li>Easier to control than tailsitter</li>
```
Comment (sfuhrer) [inline en/frames_vtol/README.md:71]:
```suggestion
    <li>Easier to control in hover than tailsitters due to more control authority</li>

```

File: en/frames_vtol/README.md [hunk 8]
```diff
@@ -1,20 +1,156 @@
...
+    <li>Can hover</li>
```
Comment (sfuhrer) [inline en/frames_vtol/README.md:72]:
```suggestion

```

File: en/frames_vtol/README.md [hunk 9]
```diff
@@ -1,20 +1,156 @@
...
+<div class="grid_item">
+  <div class="grid_item_heading">Standard VTOL</div>
+  <div class="grid_text">
+  Separate rotors/flight controls for multicopter and forward flight. Takes off and lands on belly.
+  <img src="../../assets/airframes/vtol/vertical_technologies_deltaquad/hero_small.png" title="Vertical Technologies: Deltaquad" />
+  <ul>
+    <li>Most complex. More things to go wrong but better redundancy.</li>
+    <li>Easiest to control</li>
+    <li>Can hover</li>
+    <li>Fuel engines can be used</li>
```
Comment (sfuhrer) [inline en/frames_vtol/README.md:86]:
```suggestion
    <li>Fuel engines for forward flight propulsion can be used</li>

```

File: en/frames_vtol/README.md [hunk 10]
```diff
@@ -1,20 +1,156 @@
...
+  </div>
+</div>
+
+<div class="grid_item">
+  <div class="grid_item_heading">Standard VTOL</div>
+  <div class="grid_text">
+  Separate rotors/flight controls for multicopter and forward flight. Takes off and lands on belly.
+  <img src="../../assets/airframes/vtol/vertical_technologies_deltaquad/hero_small.png" title="Vertical Technologies: Deltaquad" />
+  <ul>
+    <li>Most complex. More things to go wrong but better redundancy.</li>
```
Comment (sfuhrer) [inline en/frames_vtol/README.md:83]:
```suggestion
    <li>Additional weight from separate hover/forward flight propulsion systems</li>

```

File: en/frames_vtol/README.md [hunk 11]
```diff
@@ -1,20 +1,156 @@
...
+    <li>Easiest to control</li>
```
Comment (sfuhrer) [inline en/frames_vtol/README.md:84]:
```suggestion
    <li>Easiest to control due to dedicated hover/forward flight actuators</li>

```

File: en/frames_vtol/README.md [hunk 12]
```diff
@@ -1,20 +1,156 @@
...
+    <li>Can hover</li>
```
Comment (sfuhrer) [inline en/frames_vtol/README.md:85]:
```suggestion
    <li>Add possibility for actuator redundancy</li>

```

File: en/frames_vtol/README.md [hunk 13]
```diff
@@ -1,20 +1,156 @@
...
+
+## Assembly
+
+:::note
+For information about commercial and kit VTOL vehicles see: [Complete Vehicles](../complete_vehicles/README.md) 
+:::
+
+PX4 controlled vehicles generally share the same core components: a flight controller connected to a power system, GPS, external compass (highly recommended), radio control system (optional) and/or telemetry radio system (optional), and airspeed sensor (highly recommended for VTOL vehicles).
+
+The flight controller outputs are connected to the vehicle motor ESCS and/or flight control servos and actuators, which are separately powered.
```
Comment (sfuhrer) [inline en/frames_vtol/README.md:117]:
```suggestion
The flight controller outputs are connected to the vehicle motor ESCs and/or flight control servos and actuators, which are separately powered.

```

File: en/frames_vtol/README.md (hunk 1 above)
Comment (sfuhrer) [reply en/frames_vtol/README.md:46]: >It looks like this

Love it!

**Review by sfuhrer — COMMENTED** [review]
Thanks for the push @hamishwillee . And sorry for being so picky at the VTOL pro/cons, couldn't stop myself once started.

File: en/frames_vtol/README.md (hunk 2 above)
Comment (hamishwillee) [reply en/frames_vtol/README.md:52]: Much better!

**Comment by hamishwillee** [comment]
@sfuhrer All the suggestions are improvements and I've accepted them as is :-). Thanks so much - just the expert review that was needed.

**Pull request merged**